run_migration.py

```diff
@@ -0,0 +1,4 @@
+# TODO: make this dynamic so you can pass in the migration you want to run from the command line?
+from migrations import sample_timestamps
+
+sample_timestamps.run()
```

Comment:
Good, we shouldn't have too many more migrations, but if that is the case we could use a mongo migration framework for python